test/model/compose_test.dart

```diff
@@ -334,6 +335,80 @@ hello
     });
   });
 
+  test('fallbackMarkdownLink', () {
+    final store = eg.store();
+    final channels = [
+      eg.stream(streamId: 1, name: '`code`'),
+      eg.stream(streamId: 2, name: 'score > 90'),
+      eg.stream(streamId: 3, name: 'A*'),
+      eg.stream(streamId: 4, name: 'R&D'),
+      eg.stream(streamId: 5, name: 'UI [v2]'),
+      eg.stream(streamId: 6, name: r'Save $$'),
+    ];
+    store.addStreams(channels);
```

Comment:
This should be `await`ed; similarly in a few other places.

Thanks to the `discarded_futures` lint for catching this, actually; I was playing with it for #731 this morning 🙂